pkg/web/apps/apps_service.go

```diff
@@ -74,7 +74,7 @@ func (a *appsService) UpdateAppVersions(versions []models.Version) error {
 func (a *appsService) DisableAllAppVersionsByAppID(id string, message string) error {
 
 	// get the app id to send it to the re
-	app, err := a.repository.GetAppByID(id)
+	app, err := a.repository.GetActiveAppByID(id)
 
```

Comment:
Hi @davidffrench just to confirm that are you ok with this point? 
After this change the /apps/{id}/versions/disable endpoint will just work for the apps which are active. I mean it will not work with those ones which have the `deleted_at != NULL`. 

@austincunningham and @davidffrench , it is not a blocker to merge this PR at all, I just would like to highlight this for @davidffrench. Because if we would like to keep the endpoint `disable all versions` working for all apps and not just the active ones then we can just to create a small task to use here a method which GetAppByID without a filter by active ones. Anyway, of this change shows not go against the current requirements as well.

Reply:
makes sense

Reply:
Hey @camilamacedo86 . The `/apps/{id}/versions/disable` should only work for apps that are not soft deleted in my opinion. Soft deleted apps from a users point of view are deleted.